invenio_deposit/api.py

```diff
@@ -341,7 +341,7 @@ def publish(self, pid=None, id_=None):
             self._publish_new(id_=id_)
         else:  # Update after edit
             record = self._publish_edited()
-            record.commit()
+            record.commit(**kwargs)
```

Comment:
I'll remove this change on `publish()` (it's not really needed). 

p.s. I'm trying to write some tests but I need `invenio-records` is released with latest changes.
Could you please release it while I'm finish to create the tests? Thanks! :+1: